dvc/stage.py

```diff
@@ -129,6 +131,43 @@ def __init__(self, missing_files):
         super(MissingDataSource, self).__init__(msg)
 
 
+def rwlocked(read=[], write=[]):
```

Comment:
`@decorator` maybe?)

Reply:
@Suor Not sure I understand, could you elaborate, please? The `read` and `write ` are attr names for the stage and stage is only passed to the method itself. Seems like using `@decorator` is not really suitable here. Or am I missing something?